src/Clock/FixedClock.php

```diff
@@ -16,8 +16,9 @@ final class FixedClock implements Clock
     /**
      * @param Instant $instant The time to set the clock at.
      */
-    public function __construct(private Instant $instant)
-    {
+    public function __construct(
+        private Instant $instant
```

Comment:
Looks like we need to configure [TrailingCommaInMultilineFixer](https://cs.symfony.com/doc/rules/control_structure/trailing_comma_in_multiline.html) to work not just on arrays!

Reply:
I did it but decided not to commit as I wasn't sure you'd wanted the change. I could do it here or in another PR, as you'd like.

Reply:
Here is fine!